packages/discovery-provider/plugins/notifications/src/config.ts

```diff
@@ -7,6 +7,8 @@ export const config = {
   blastUserBatchSize: 100,
   // Batch size of notifications to be processed together
   notificationBatchSize: 30,
+  // Only process blasts older than this delay (in seconds)
```

Comment:
Maybe worth spelling out *why* as it's probably not going to be obvious down th eroad